### PR TITLE
Added configuration / converters for LXX60-CS27LX1.0 and LXN59-CS27LX1.0

### PR DESCRIPTION
**Pull request by a-bailey**
Added configuration for 2 versions of this cover: https://www.amazon.de/gp/product/B08BFL1HDN/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

### CODE REVIEW

File: devices/tuya.js [hunk 1]
```diff
@@ -740,6 +740,26 @@ module.exports = [
             exposes.enum('backlight_mode', ea.ALL, ['LOW', 'MEDIUM', 'HIGH']),
             exposes.numeric('calibration_time', ea.STATE).withUnit('S').withDescription('Calibration time')],
     },
+    {
+        zigbeeModel: ['LXN59-CS27LX1.0'],
+        model: 'LXN59-CS27LX1.0',
+        vendor: '3A Smart Home DE',
```
Comment (Koenkk) [inline devices/tuya.js:746]: Please move this to `neu_3a.js`

File: converters/fromZigbee.js [hunk 2]
```diff
@@ -1517,6 +1517,28 @@ const converters = {
...
+            }
+            return result;
+        },
+    },
+    LXN59_cover_state_via_onoff: {
+        cluster: 'genOnOff',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.hasOwnProperty('onOff')) {
+                return {state: msg.data['onOff'] === 1 ? 'CLOSED' : 'OPEN'};
```
Comment (Koenkk) [inline converters/fromZigbee.js:1538]: `CLOSED` should be `CLOSE` (similar to other covers)

Comment (a-bailey) [reply converters/fromZigbee.js:1538]: ah sure you are right will correct that

File: devices/tuya.js (hunk 1 above)
Comment (a-bailey) [reply devices/tuya.js]: Done

File: converters/fromZigbee.js (hunk 2 above)
Comment (a-bailey) [reply converters/fromZigbee.js]: Done

File: devices/nue_3a.js [hunk 3]
```diff
@@ -247,6 +273,40 @@ module.exports = [
...
+            await reporting.currentPositionLiftPercentage(endpoint1);
+            device.powerSource = 'Mains (single phase)';
+            device.save();
+        },
+    },
+    {
+        zigbeeModel: ['LXX60-CS27LX1.0'],
+        model: 'ZW-EU-02',
+        vendor: 'Nue / 3A',
+        description: 'ZigBee smart curtain switch',
```
Comment (Koenkk) [inline devices/nue_3a.js:299]: @a-bailey is this really a curtain switch? Or is it the 1 channel switch from the Amazon page (?)

**Comment by a-bailey** [comment]
@Koenkk it is the curtain switch but sadly a total mess I ordered 3 times and everytime the device behaved completely different that's why I have 2 implementations for the "same" module. I couldn't get stop to work for the first one but luckily I ordered most of my devices in 1 bulk so at least they are all the same

**Comment by a-bailey** [comment]
sorry on the other one its ZW-EC-01  ill change it

**Comment by a-bailey** [comment]
> What is the model number for both on the back of the device, `ZW-EU-4C`?

ZW-EC-01 on the other one I'll change it

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**